components/stacks-network/src/orchestrator.rs

```diff
@@ -1452,6 +1488,7 @@ events_keys = ["*"]
 
         let options = CreateContainerOptions {
             name: format!("subnet-node.{}", self.network_name),
+            platform: Some(devnet_config.docker_platform.to_string()),
```

Comment:
Is there a reason we're adding this `docker_platform` field as part of this PR? Does this impact anything?

Reply:
This is now required. It's because of the bollard update.
It's probably related to the issue Brice had, but also, the image he was using (the one I built) is probably badly configured (#1305)

I'll double check this docker_platform behavior, it's not behaving as I expected it to